script.js

```diff
@@ -44,4 +56,199 @@ hamburgerBtn.addEventListener('click', displayMobileMenu);
 cancelBtn.addEventListener('click', reverseEvent);
 for (let i = 0; i < menuElements.length; i += 1) {
   menuElements[i].addEventListener('click', chooseSection);
-}
\ No newline at end of file
+}
+
+// Project detail popup window
+// store popup window data
+const projectDetail = [
+  {
+    id: 1,
+    name: 'Keeping track of hundreds  of components website',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featured_image: 'images/featured-images/project-img-1.png',
+    technologies: {
+      0: 'html',
+      1: 'Bootstrap',
+      2: 'Ruby on rails',
+    },
+    linkTo_live: '#',
+    linkTo_source: '#',
+  },
+  {
```

Comment:
- [ ] [**OPTIONAL**] When you want to see a project you are supposed to see the same description, the same picture, the same project title. Kindle make sure you correct that ! :+1: 

### Before click

![Screenshot from 2023-01-06 20-03-29](https://user-images.githubusercontent.com/80458199/211071843-7ad8b2c9-98b1-4dd4-b8d9-c08eaabc5679.png)


### Popup After click

![Screenshot from 2023-01-06 20-03-46](https://user-images.githubusercontent.com/80458199/211071885-2f6fc9da-fc79-4bb3-b3c8-1540f93487c3.png)